apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -1579,12 +1579,20 @@ public static Bundle addResourcesToBundle(Bundle bundle, List<IBaseResource> res
     return bundle;
   }
 
+  /** @param beneIds the {@link Collection} beneficiary IDs to log */
```

Comment:
Missing description on this and the one below (one day I'll get checkstyle enabled for the rest of these...)

Reply:
done